src/vt/vrt/collection/balance/node_lb_data.h

```diff
@@ -249,6 +249,13 @@ struct NodeLBData : runtime::component::Component<NodeLBData> {
    */
   LBDataHolder* getLBData() { return lb_data_.get(); }
 
+  /**
+   * \internal \brief Set the minimal amount of historical LB data which should be hold
```

Comment:
```suggestion
   * \internal \brief Set the minimal amount of historical LB data which should be held
```

Reply:
Or maybe even better:
```suggestion
   * \internal \brief Set the minimal amount of historical LB data which should be retained
```
?

Reply:
Whichever option you choose, the same applies to line 255 below.